sn_networking/src/event.rs

```diff
@@ -234,6 +302,16 @@ impl SwarmDriver {
                 }
             },
             SwarmEvent::NewListenAddr { address, .. } => {
+                if address.iter().any(|p| p == libp2p::multiaddr::Protocol::P2pCircuit) {
+                    info!("Relayed listen address added: {}", address.clone());
```

Comment:
lets make this a Marker

Reply:
Which is more helpful here?
RelayedListenAddressAdded
RelayReservationAccepted

And should I add another for RelayedConnectionEstablished in the ConnectionEstablished event?